doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -723,8 +723,8 @@ The following table indicates the software maturity levels of the support for ea
              - Supported
              - Experimental
            * - **Channel Sounding**
-             - --
-             - --
+             - Supported
+             - Supported
              - Supported
              - Experimental
```

Comment:
Should it be Supported instead of Experimental for nRF54LM20 as per the commit message?